### DISCUSSION TITLE
How to make horizontal scroll

### DISCUSSION THREAD

**Discussion by Devf1s** [darkroomengineering/lenis#79] [Q&A]
Maybe that's a bit of a stupid question to ask how it works, but unfortunately, there's very little information about this documentation on the internet, therefore, I'm writing here: **How to make a horizontal scroll using Lenis?** I don't see any example of it. Does it use third-party libraries in [demo](https://lenis.studiofreight.com/) site?

**Comment by winstonbedwar** [comment]
hiya, do you have any update on this? i'm interested in learning more about this as well. on my end, even when i set position: sticky; it ignores it entirely and scrolls past.

**Comment by Devf1s** [comment]
I don’t remember to be honest, I completed the order so I think everything is fine. Check my [repository ](https://github.com/1KiritoS1/bine-games)( **roadmap section** )
